packages/core/src/components/graphs/area-stacked.ts

```diff
@@ -73,7 +72,9 @@ export class StackedArea extends Component {
 					originalClassName: 'area'
 				})
 			)
-			.style('fill', (d: any) => self.model.getFillColor(getProperty(d, 0, groupMapsTo)))
+			.style('fill', (d: any) =>
+				self.model.getFillColor(getProperty(d, 0, groupMapsTo), undefined, d)
```

Comment:
we shouldn't send `undefined` over.

Here's an example in `scatter.ts`
```js
return this.model.getFillColor(d[groupMapsTo], d[domainIdentifier], d)
```

Let's try to send all of that info over pls, and in case we can't locate the data, use `null` instead of `undefined` pls

Reply:
This applies to all the files in the PR

Reply:
[https://charts.carbondesignsystem.com/documentation/classes/Chart.html](url)
the charts in this list don't have access to cartesianScale hence I couldn't use getDomainIdentifier with those and some of the charts receive binned and stacked data. I'll be sending null as mentioned.